spec/components/candidate_interface/degree_required_component_spec.rb

```diff
@@ -21,6 +21,32 @@
     end
   end
 
+  context 'application has uk degree and cbut degree grade marked as not_required' do
+    let(:course_option1) { create(:course_option, course: create(:course, :open_on_apply, degree_grade: 'not_required')) }
```

Comment:
Since there is no other `course_option` used in this context, can we rename `course_option1` -> `course_option`
```suggestion
    let(:course_option) { create(:course_option, course: create(:course, :open_on_apply, degree_grade: 'not_required')) }
```

Reply:
Yep np, a hangover from a previoius commit